deploy/scripts/src/mobile/backend-e2e.test.ts

```diff
@@ -25,6 +25,20 @@ const profiles: ProfileList = {
       ],
       exec: 'backendJourney'
     }
+  },
+  load: {
+    backendJourney: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1700, // Calculation: 100 journeys / second * 17 seconds average journey time
+      maxVUs: 3000, // Calculation: 100 journeys / second * 2.5 seconds maximum expected from NFR (2.5 per request, 10 user-facing requests + safety)
+      stages: [
+        { target: 100, duration: '15m' }, // linear increase from 0 iteration per second to 100 iterations per second for 15 min -> 0.11 t/s/s
```

Comment:
```suggestion
      preAllocatedVUs: 1000, // Calculation: 100 journeys / second * 10 seconds average journey time
      maxVUs: 3000, // Calculation: 100 journeys / second * 2.5 seconds maximum expected from NFR (2.5 per request, 10 user-facing requests + safety)
      stages: [
        { target: 100, duration: '15m' }, // linear increase from 1 iteration per second to 100 iterations per second for 15 min -> 0.11 t/s/s
```
Suggest lowering pre-allocated VUs to base it on a 10 second average journey time.

Also just updated comment on start rate - this starts at 1/s rather than 0.

Reply:
Thanks, @Tom-Dann, yes our average duration is 10s.